gnd/src/main/java/com/google/android/gnd/Config.java

```diff
@@ -30,4 +30,7 @@
 
   // Photos
   public static final String PHOTO_EXT = ".jpg";
+
+  // Local geojson source settings.
+  public static final String GEO_JSON_SOURCE = "gnd-geojson.json";
```

Comment:
I went with `source` because it's possible that we'll use a URL in the future instead of a file on the device...maybe just `GEO_JSON` would suffice with a more detailed comment?

Reply:
e4a770b -- went with GEO_JSON and added additional commentary.